src/select/SelectListState.ts

```diff
@@ -14,15 +14,11 @@ import {
 export function useSelectListState(
   initialState: SealedInitialState<SelectListInitialState> = {},
 ): SelectListStateReturn {
-  const {
-    currentId = null,
```

Comment:
Thats because I removed this !

Now by default current id will be on the first tabbable element as programmed in composite.

Reply:
This will only affect the currentValue not the selectedValue.

If the selectedValue is changed, currentId will change on useEffect when visible
![image](https://user-images.githubusercontent.com/39694575/99790963-e6854500-2b4a-11eb-9a2c-30c84c0568c5.png)

Reply:
Got it!